src/commands/subcommands/community/handleSuggest.ts

```diff
@@ -35,9 +35,10 @@ export const handleSuggest: CommandHandler = async (
       return;
     }
 
-    const suggestionChannel = guild.channels.cache.find(
-      (el) => el.id === config.suggestion_channel
-    ) as TextChannel;
+    const suggestionChannel =
+      (guild.channels.cache.find(
+        (el) => el.id === config.suggestion_channel
+      ) as TextChannel) || ForumChannel;
```

Comment:
```suggestion
    const suggestionChannel =
      guild.channels.cache.find(
        (el) => el.id === config.suggestion_channel
       as TextChannel) | ForumChannel;
```

This is not correct. The code is currently saying `suggestionChannel` should be:
- The channel resolved from the cache as `TextChannel` OR
- The value of `ForumChannel`.

We need to use `|`, which tells TypeScript that `suggestionChannel` is either the type `TextChannel` or `ForumChannel`.

This **will** create compiler errors, because forum channels have different methods than text based channels, and as such the logic further down will need to be updated.

Reply:
That makes more sense... the compiler errors threw me off